src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -150,6 +151,16 @@ public function reboot($warmupDir)
         $this->boot();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function reset()
```

Comment:
we could also remove this method and move the implementation to Application::reset

Reply:
I think it make sens, since we have reboot()? I don't have a strong opinion on this one :)

Reply:
Unless we have a use case, let's not add it: on the web, this is already dealt with during reboot - and on the console, this is on Application. All good to me.